src/config.rs

```diff
@@ -708,6 +708,20 @@ impl<'a> Cfg<'a> {
         }
     }
 
+    #[cfg_attr(feature = "otel", tracing::instrument)]
+    pub(crate) async fn rustc_version(&mut self) -> Result<String> {
```

Comment:
I got a bit confused between `Cfg::rustc_version` and `Toolchain::rustc_version` when reviewing this and #3881 at the same time (x_x)

Maybe rename it to `Cfg::active_rustc_version` for clarity?

Reply:
Fair, done.